tests/features/layers/activation-order/sketch.ino

```diff
@@ -0,0 +1,98 @@
+/* -*- mode: c++ -*-
+ * Copyright (C) 2020  Keyboard.io, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "Kaleidoscope.h"
+
+#include "Kaleidoscope-Macros.h"
+
+// *INDENT-OFF*
+
+KEYMAPS(
+  [0] = KEYMAP_STACKED
+  (
+    Key_A ,Key_A ,Key_A ,Key_A ,Key_A ,Key_A ,Key_A
+   ,Key_A ,Key_A ,Key_A ,Key_A ,Key_A ,Key_A ,Key_A
```

Comment:
Just for clarity, I think it probably makes sense to only have keys we -need- defined on the layer. so just one Key_A.

Reply:
And -maybe- to have it be Key_0 to represent the layer 0? Not sure on that one.

Reply:
`Key_0` makes sense. I went with full-`Key_A` (except for the layer shifts and the macro) to have a uniform layout. But having all the unused keys blank works just as well, I suppose.

Reply:
If you want to use a single sketch file for as many testcases as possible (or optimal, given readability considerations), it would be good to have layer-shift keys that are mapped to other keys on the target layer. I think it's reasonable to have some keys transparent, and others (not just one) identify the layer number.